config/test.exs

```diff
@@ -9,7 +9,8 @@ config :screens, ScreensWeb.Endpoint,
 config :screens,
   config_fetcher: Screens.Config.State.LocalFetch,
   local_config_file_spec: {:test, "config.json"},
-  signs_ui_config_fetcher: Screens.SignsUiConfig.State.LocalFetch
+  signs_ui_config_fetcher: Screens.SignsUiConfig.State.LocalFetch,
+  default_api_v3_url: [:no_api_requests_allowed_during_testing]
```

Comment:
Consider adding a comment above this explaining the reasoning, and/or demonstrating how to pass a `fetch_*` function in as an argument with a default value.

Main points:
- Functions that request data from an external resource outside our control cause flaky tests
- Pass request-firing functions in as arguments with the "normal" fetch function as the default value, so that we can easily pass a stubbed version in tests